webrender/src/device/gfx/device.rs

```diff
@@ -1258,7 +1258,8 @@ impl<B: hal::Backend> Device<B> {
             passes: hal::pass::SubpassRef::External .. hal::pass::SubpassRef::Pass(0),
```

Comment:
the external barriers shouldn't be necessary any more if you use the specific image layouts

Reply:
Fixed them: https://github.com/szeged/webrender/pull/311/commits/b6f19147a6d19c5a9fd053354ceb11c505ab4b9e#diff-cdf03c96bd34cb98c2d09b0be7658232R1258